src/Bicep.Core/Registry/OciModuleRegistry.cs

```diff
@@ -298,12 +269,18 @@ public static string GetPublicBicepModuleDocumentationUri(string publicModuleNam
 
         public override async Task PublishArtifact(OciModuleReference moduleReference, Stream compiled, string? documentationUri, string? description)
         {
+            // Write out an empty config for now
+            // NOTE: Bicep v0.20 and earlier will throw if it finds a non-empty config
```

Comment:
>Bicep v0.20 [](http://example.com/codeflow?start=21&length=11)

Do we need to call this out in release notes to warn users?

Reply:
It's not a breaking change yet, preparation for breaking change (https://github.com/Azure/bicep/issues/11482).

@majastrz @shenglol 
We had discussed whether we should add a warning letting them know we expect a breaking change.  I'm open to this, although uncertain what we could say that would be useful and where/when to do that. Ideas?